tests/exception/bulkwriteexception-getresultdocument-001.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+MongoDB\Driver\Exception\BulkWriteException write result
```

Comment:
This can be: `MongoDB\Driver\Exception\BulkWriteException::getWriteResult()`

Also, the filename should be `bulkwriteexception-getwriteresult-001.phpt`.